build/builder.sh

```diff
@@ -134,4 +134,6 @@ docker run -i ${tty-} ${rm} \
   --env="CIRCLE_NODE_INDEX=${CIRCLE_NODE_INDEX-0}" \
   --env="CIRCLE_NODE_TOTAL=${CIRCLE_NODE_TOTAL-1}" \
   --env="GOTRACEBACK=${GOTRACEBACK-all}" \
+  --env="COVERALLS_TOKEN=${COVERALLS_TOKEN}" \
```

Comment:
unless these environment variables are _always_ set in the calling shell, we're gonna need a default.